maps/away/iceland/iceland.dm

```diff
@@ -0,0 +1,22 @@
+/datum/away_map/iceland
+	name = "iceland"
+	full_name = "Ice Land"
+	path = "iceland"
+	weathertypes = list("ice", "hail")
+
+/datum/away_map/iceland/handle_random_gen()
+	new /datum/random_map/noise/tundra(rand(2,99999), 2, 2, zloc, 44, 44)
```

Comment:
Doesn't this automatically set a seed? Explicitly setting this is causing the random map to bypass lag-checks (because for some reason that's tied to seed).